Patch yarnrc

This is important because in the past we've had issues with the yarnrc causing native module errors while developing and running code-server. 

We patch it by ensuring that the version of node used is always the locally installed one.
Index: quilt-poc/node.js
===================================================================
--- quilt-poc.orig/node.js
+++ quilt-poc/node.js
@@ -7,7 +7,11 @@ Object.defineProperty(exports, "__esModu
 const path = require("path");
 const fs = require("fs");
 const root = path.dirname(path.dirname(__dirname));
-const version = /^target\s+"([^"]+)"$/m.exec(yarnrc)[1];
+// NOTE@coder: Always use the current version to avoid native module errors.
+// const yarnrcPath = path.join(root, 'remote', '.yarnrc');
+// const yarnrc = fs.readFileSync(yarnrcPath, 'utf8');
+// const version = /^target\s+"([^"]+)"$/m.exec(yarnrc)[1];
+const version = process.versions.node;
 const platform = process.platform;
 const arch = platform === 'darwin' ? 'x64' : process.arch;
 const node = platform === 'win32' ? 'node.exe' : 'node';
